sonarr/qualitydefinition.go

```diff
@@ -0,0 +1,62 @@
+package sonarr
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"strconv"
+)
+
+// GetQualityDefinitions returns all configured quality definitions.
+func (s *Sonarr) GetQualityDefinitions() ([]*QualityDefinition, error) {
+	return s.GetQualityDefinitionsContext(context.Background())
+}
+
+func (s *Sonarr) GetQualityDefinitionsContext(ctx context.Context) ([]*QualityDefinition, error) {
+	var definitions []*QualityDefinition
+
+	if _, err := s.GetInto(ctx, "v3/qualityDefinition", nil, &definitions); err != nil {
+		return nil, fmt.Errorf("api.Get(qualityDefinition): %w", err)
+	}
+
+	return definitions, nil
+}
+
+// GetQualityDefinition returns a single quality definition.
+func (s *Sonarr) GetQualityDefinition(qualityDefinitionID int) (*QualityDefinition, error) {
+	return s.GetQualityDefinitionContext(context.Background(), qualityDefinitionID)
+}
+
+func (s *Sonarr) GetQualityDefinitionContext(ctx context.Context, qualityDefinitionID int) (*QualityDefinition, error) {
+	var definition *QualityDefinition
```

Comment:
I'm not super picky about this, but I like to name the output `output`.  It's a little shorter and keeps consistency across methods.
```
var output *QualityDefinition
```

Reply:
You're right it's clearer this way! just changed!